deployer/src/config_creator.py

```diff
@@ -113,6 +119,28 @@ def to_vuepress_config(config):
     return config
 
 
+def to_larecipe_config(config, urls=None):
+    if urls:
+        config["sitemap_urls"] = [
+            extract_root_from_input(urls[0]) + "sitemap.xml"]
+    config["selectors"]["lvl0"] = OrderedDict((
```

Comment:
Question: why do we need to use an `OrderedDict`? Also, is there a reason we need to define each subkey one by one?

Would it work the same with the following syntax (still new to python, wondering if there is something I miss here):

```
    config["selectors"] = {
        "lvl0": {
            "selector": "//div[contains(@class, 'sidebar')]//li/a[text()=//div[contains(@class, 'article')]//h1[1]/text()]",
            "global": True,
            "type": "xpath",
            "default_value": "Documentation",
        },
        "lvl1": "div.article h1",
        "lvl2": "div.article h2",
        "lvl3": "div.article h3",
        "lvl4": "div.article h4",
        "lvl5": "div.article h5",
        "text": "div.article p, div.article li",
    }
```

Reply:
Yes we need both. The re is no other way? We wan order to have consistency among files

Reply:
The correct way if you want to keep the order, is to use the [update()](https://docs.python.org/3/library/stdtypes.html#dict.update) method (According to the Python doc, it keep the order of the entries - maybe I'm wrong).
So something like this:
```python
selectors = {
   'lvl0': {...},
   'lvl1': 'div.article h1',
}
config["selectors"].update(selectors)
```
WDYT?

Reply:
Good point about keeping the order in the final config. This makes diffing easier, so let's keep it.

If `update` indeed keep the order, this will be much easier to read and avoid bad copy-paste to have it presented as a hash like this.